cibuildwheel/__main__.py

```diff
@@ -400,7 +404,7 @@ def get_build_identifiers(
         List[cibuildwheel.macos.PythonConfiguration],
     ]
 
-    if platform == "linux":
+    if platform == "linux" or platform == "crosslinux":
```

Comment:
```suggestion
    if platform in {"linux", "crosslinux"}:
```

Minor, but CPython optimizes this so it's at least as fast as the other expression, and it reads better.

Reply:
Left as it is to satisfy the mypy.